plugins/fluentd_telemetry_plugin/README.md

```diff
@@ -192,24 +192,27 @@ Usage
-|    [streaming.stream_only_new_samples](conf/fluentd_telemetry_plugin.cfg#L16)    |   True   |                                                                                                    If True, the plugin will stream only the changed values [Default is True]                                                                                                     |
-|            [streaming.enabled](conf/fluentd_telemetry_plugin.cfg#L17)            |   True   |                                                                                     If True, the streaming will be started once the required configurations have been set [Default is False]                                                                                     |
-|       [logs-config.logs_file_name](conf/fluentd_telemetry_plugin.cfg#L20)        |   True   |                                                                                                                     Log file name [Default = '/log/tfs.log']                                                                                                                     |
-|         [logs-config.logs_level](conf/fluentd_telemetry_plugin.cfg#L22)          |   True   |                                                                                                                                Default is 'INFO'                                                                                                                                 |
-|      [logs-config.max_log_file_size](conf/fluentd_telemetry_plugin.cfg#L24)      |   True   |                                                                                                                Maximum log file size in Bytes [Default is 10 MB]                                                                                                                 |
-|    [logs-config.log_file_backup_count](conf/fluentd_telemetry_plugin.cfg#L26)    |   True   |                                                                                                                Maximum number of backup log files [Default is 5]                                                                                                                 |
+|                                         Parameter                                         | Required |                                                                                                                                   Description                                                                                                                                    |
+|:-----------------------------------------------------------------------------------------:|:--------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
```

Comment:
I think you should have a separate column for the default value

Reply:
I agree, handled